index.html

```diff
@@ -721,6 +721,18 @@ <h4>Verify Presentation</h4>
           data-api-endpoint="post /presentations/verify"></div>
       </section>
 
+      <section>
+        <h4>Create Challenge</h4>
+        <p>
+        </p>
+
+        <div class="api-detail"
+          data-api-endpoint="post /challenges"></div>
+        <p>
+The issuer should track the number of times a challenge has been passed to an endpoint as `options.challenge`.
```

Comment:
Hmm... this isn't quite right, some quick changes here that might need more work:

```suggestion
The instance should create a challenge for use during verification and it should keep track of the number of times a challenge has been passed to verification endpoints as `options.challenge`.
```

Reply:
Maybe?
```suggestion
The instance should create a challenge for use during verification, and track the number of times each challenge has been passed to verification endpoints as `options.challenge`.
```